<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenhuaiFerControlTabPageWidget</class>
 <widget class="QWidget" name="RenhuaiFerControlTabPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1730</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="loginHistoryFrame">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>460</y>
     <width>455</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 222, 222);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="loginHistoryFrame_horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QFrame" name="alertHistoryFrame">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>460</y>
     <width>445</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 222, 222);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="alert_history_horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QFrame" name="state_frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>460</y>
     <width>810</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 222, 222);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="state_frame_horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QFrame" name="fer_tank_frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>1721</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 222, 222);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="group_one_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>480</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="group_one_frame_horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QPushButton" name="fanPushButton_1">
     <property name="geometry">
      <rect>
       <x>202</x>
       <y>340</y>
       <width>64</width>
       <height>85</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>324</y>
       <width>345</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="src.qrc">:/image/new/horizon_pipe.bmp</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="group_two_frame">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>480</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="group_two_frame_horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QPushButton" name="fanPushButton_2">
     <property name="geometry">
      <rect>
       <x>202</x>
       <y>340</y>
       <width>64</width>
       <height>85</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>324</y>
       <width>345</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="src.qrc">:/image/new/horizon_pipe.bmp</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="group_three_frame">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>320</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="group_three_frame_horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QPushButton" name="fanPushButton_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>340</y>
       <width>64</width>
       <height>85</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>324</y>
       <width>245</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="src.qrc">:/image/new/horizon_pipe.bmp</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="group_four_frame">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>10</y>
      <width>320</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="group_four_frame_horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QPushButton" name="fanPushButton_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>340</y>
       <width>64</width>
       <height>85</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>324</y>
       <width>245</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="src.qrc">:/image/new/horizon_pipe.bmp</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
